transformers/dart-dvhcvn/lib/src/model.dart

```diff
@@ -48,7 +48,7 @@ abstract class Entity<ParentType, ChildType> {
 
   @override
   String toString() {
-    final parent = this.parent?.toString();
+    final String? parent = this.parent?.toString();
```

Comment:
Let's omit the type here for cleaner code. This is just a personal preference.

```suggestion
    final parent = this.parent?.toString();
```